File: LeetCode_Solutions/3110.py
def main(s):
    x = 0
    y = 1
    res = 0

    while y < len(s):
        val = abs((ord(s[x])) - ord(s[y]))
        x += 1
        y += 1
        res += val

    print(res)

main('hello')

File: LeetCode_Solutions/3110.c
#include <string.h>
#include<stdlib.h>

int scoreOfString(char* s) {
    int res = 0;
    int x = 0;
    int y = 1;
    size_t i;

    while (y < strlen(s)) {
        res += abs(s[x] - s[y]);
        y++;
        x++;
}   return res; }